lib-rt/setup.py

```diff
@@ -79,8 +82,14 @@ def run(self) -> None:
     cflags: list[str] = []
     if compiler.compiler_type == "unix":
         cflags += ["-O3"]
+        if X86_64:
+            cflags.append("-msse4.2")  # Enable SIMD (see also mypyc/build.py)
```

Comment:
No auto detection? I'll have to patch this out for Debian, the amd64 baseline is sse2

Reply:
Looking at the original mypy PR https://github.com/python/mypy/pull/20244 that added this, I guess it was necessary for some compilers, maybe @JukkaL can clarify.

Reply:
The build fails without this, since SSE4.2 features can't be used without the flag, at least on Ubuntu 24.04. In particular, the C files within `arch/sse` in the path may need this flag. Auto-detection is supported by the underlying base64 library, but we'd still need to compile the SSE-related files using this flag, and setuptools doesn't make it straightforward to have different C compiler flags for specific files.